_posts/2019/2019-04-01-a-time-travel-through-our-latest-hackathon.md

```diff
@@ -0,0 +1,102 @@
+---
+layout: post
+title: "A time travel through our latest Hackathon"
+date: 2019-04-01
+header_image: public/loc-interview.jpg
```

Comment:
Björn promised to have a look but he needed to travel back to Jena. Will add this on Monday.